repositories/repositories.bzl

```diff
@@ -65,6 +65,33 @@ def ros2_repositories():
 
     ros2_repositories_impl()
 
+    maybe(
+        http_archive,
+        name = "attr",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:attr.BUILD.bzl",
+        sha256 = "25772f653ac5b2e3ceeb89df50e4688891e21f723c460636548971652af0a859",
+        url = "http://download.savannah.gnu.org/releases/attr/attr-2.4.47.src.tar.gz",
+        strip_prefix = "attr-2.4.47",
+    )
+
+    maybe(
+        http_archive,
+        name = "acl",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:acl.BUILD.bzl",
+        sha256 = "06be9865c6f418d851ff4494e12406568353b891ffe1f596b34693c387af26c7",
+        url = "https://de.freedif.org/savannah/acl/acl-2.2.53.tar.gz",
+        strip_prefix = "acl-2.2.53",
+    )
+
+    maybe(
```

Comment:
Should be used from the ros2.repos https://github.com/ros2/ros2/blob/humble-release/ros2.repos#L46. See https://github.com/mvukov/rules_ros2/blob/main/docs/ros2_repos.md; please run the resolver once you update the yaml file.

But this should happen after the target PR is consolidated.